packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -274,8 +274,6 @@ export class GCFBootstrapper {
        */
       logger.resetBindings();
       logger.addBindings(buildTriggerInfo(triggerType, id, name, request.body));
-      logger.metric(`Execution started by ${triggerType}`);
```

Comment:
Looking through our applications, this was currently the only place we explicitly call `logger.metric`, I wasn't convinced we need to keep this log line.